src/SonarScanner.MSBuild.Common/ApplicationCultureInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace SonarScanner.MSBuild.Common
+{
+    public sealed class ApplicationCultureInfo : IDisposable
```

Comment:
A small helper class that sets and resets the culture when the converter function is called. Needs some tests. Works like so:
* `using var _ = new ApplicationCultureInfo(someCulture)` sets the culture of the application (the "Culture and UICulture of all threads (new and running)) to `someCulture`. The previous set default culture is stored in two fields.
* when `Dispose` is called, the previous default cultures are reset.